modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -0,0 +1,476 @@
+
+In the Iceberg specification, tables consist of the following layers:
+
+* Data layer: Stores the data in data files. The Iceberg integration currently supports the Parquet file format. Parquet files are column-based and suitable for analytical workloads at scale. They come with compression capabilities that optimize files for object storage. 
+* Metadata layer: Stores table metadata separately from data files. The metadata layer allows multiple writers to stage metadata changes and apply updates atomically. It also supports database snapshots, and time travel queries that query the database at a previous point in time. 
++
+--
+** Manifest files: Track data files and contain metadata about these files, such as record count, partition membership, and file paths.
+** Manifest list: Tracks all the manifest files belonging to a table, including file paths and upper and lower bounds for partition fields. 
+** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
+--
++
+For Iceberg-enabled topics, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. The Iceberg integration supports two xref:manage:iceberg/use-iceberg-catalogs.adoc[catalog integration] types. You can configure Redpanda to catalog files stored in the same object storage bucket or container where the Iceberg data files are located, or you can configure Redpanda to use an https://iceberg.apache.org/terms/#decoupling-using-the-rest-catalog[Iceberg REST catalog^] endpoint to update an externally-managed catalog when there are changes to the Iceberg data and metadata.
+
+// image::shared:iceberg-integration-optimized.png[Redpanda's Iceberg integration]
+image::ROOT:shared:iceberg-integration-optimized.png[Redpanda's Iceberg integration]
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage, in addition to the log segments uploaded using Tiered Storage. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through many compatible data lakehouse, data engineering, and business intelligence https://iceberg.apache.org/vendors/[tools^]. 
+
+== Prerequisites
+
+To enable Iceberg for Redpanda topics, you must have the following:
+
+ifdef::env-byoc[]
+* A running xref:get-started:cluster-types/byoc/index.adoc[BYOC] cluster. The Iceberg integration is supported only for BYOC.
+* *rpk*: See xref:get-started:rpk-install.adoc[].
```

Comment:
```suggestion
* rpk: See xref:get-started:rpk-install.adoc[].
```
especially in the Cloud version, it looks a little odd for just rpk to be bolded